Response code 404 (Not Found)

Looks like UPX binaries [are not available](https://github.com/crazy-max/WindowsSpyBlocker/commit/9afcb25fff5d5f235be132ef3c139aba7280934b/checks?check_suite_id=390584450#step:7:1).

```
Run svenstaro/upx-action@v1-release
⬇️ Downloading UPX...
##[error]Response code 404 (Not Found)
(node:3188) UnhandledPromiseRejectionWarning: HTTPError: Response code 404 (Not Found)
    at EventEmitter.<anonymous> (d:\a\_actions\svenstaro\upx-action\v1-release\node_modules\got\index.js:482:24)
    at EventEmitter.emit (events.js:210:5)
    at getResponse (d:\a\_actions\svenstaro\upx-action\v1-release\node_modules\got\index.js:320:5)
    at Immediate.<anonymous> (d:\a\_actions\svenstaro\upx-action\v1-release\node_modules\got\index.js:147:6)
    at processImmediate (internal/timers.js:439:21)
(node:3188) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3188) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
##[error]Node run failed with exit code 1
```

--- a/lib/action.js
+++ b/lib/action.js
@@ -26,19 +26,19 @@ function downloadUpx() {
     return __awaiter(this, void 0, void 0, function* () {
         const tmpdir = fs.mkdtempSync(path.join(os.tmpdir(), "upx-action-"));
         if (os.type() == "Linux") {
-            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-16bfa7b-travis/amd64-linux-gcc-8-release/upx-git-16bfa7b846cf.out?raw=true', tmpdir, { filename: "upx" });
+            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-travis/amd64-linux-gcc-9-release/upx-git-0f4975fd7ffb.out?raw=true', tmpdir, { filename: "upx" });
             const upx_path = `${tmpdir}/upx`;
             fs.chmodSync(upx_path, "755");
             return upx_path;
         }
         else if (os.type() == "Darwin") {
-            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-16bfa7b-travis/amd64-darwin-clang-1000-release/upx-git-16bfa7b846cf.out?raw=true', tmpdir, { filename: "upx" });
+            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-travis/amd64-darwin-clang-1100-release/upx-git-0f4975fd7ffb.out?raw=true', tmpdir, { filename: "upx" });
             const upx_path = `${tmpdir}/upx`;
             fs.chmodSync(upx_path, "755");
             return upx_path;
         }
         else if (os.type() == "Windows_NT") {
-            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-570b2d0-appveyor/amd64-win64-msvc-14.1-release/upx-git-570b2d0e88d1.exe?raw=true', tmpdir, { filename: "upx.exe" });
+            yield download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-appveyor/amd64-win64-msvc-14.1-release/upx-git-0f4975fd7ffb.exe?raw=true', tmpdir, { filename: "upx.exe" });
             const upx_path = `${tmpdir}/upx.exe`;
             fs.chmodSync(upx_path, "755");
             return upx_path;

--- a/src/action.ts
+++ b/src/action.ts
@@ -8,17 +8,17 @@ import * as path from 'path';
 async function downloadUpx(): Promise<string> {
     const tmpdir = fs.mkdtempSync(path.join(os.tmpdir(), "upx-action-"));
     if (os.type() == "Linux") {
-        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-16bfa7b-travis/amd64-linux-gcc-8-release/upx-git-16bfa7b846cf.out?raw=true', tmpdir, { filename: "upx" });
+        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-travis/amd64-linux-gcc-9-release/upx-git-0f4975fd7ffb.out?raw=true', tmpdir, { filename: "upx" });
         const upx_path = `${tmpdir}/upx`;
         fs.chmodSync(upx_path, "755");
         return upx_path;
     } else if (os.type() == "Darwin") {
-        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-16bfa7b-travis/amd64-darwin-clang-1000-release/upx-git-16bfa7b846cf.out?raw=true', tmpdir, { filename: "upx" });
+        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-travis/amd64-darwin-clang-1100-release/upx-git-0f4975fd7ffb.out?raw=true', tmpdir, { filename: "upx" });
         const upx_path = `${tmpdir}/upx`;
         fs.chmodSync(upx_path, "755");
         return upx_path;
     } else if (os.type() == "Windows_NT") {
-        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20190303-570b2d0-appveyor/amd64-win64-msvc-14.1-release/upx-git-570b2d0e88d1.exe?raw=true', tmpdir, { filename: "upx.exe" });
+        await download.default('https://github.com/upx/upx-automatic-builds/blob/devel-20200108-0f4975f-appveyor/amd64-win64-msvc-14.1-release/upx-git-0f4975fd7ffb.exe?raw=true', tmpdir, { filename: "upx.exe" });
         const upx_path = `${tmpdir}/upx.exe`;
         fs.chmodSync(upx_path, "755");
         return upx_path;

